main.js

```diff
@@ -62,8 +68,47 @@ function deselectButton(activity) {
 }
 
 function startActivityFunc() {
-  var activity = new Activity(currentActivity, activityTask.value, activityMinutes.value, activitySeconds.value, false);
-  displayTimerPage();
+  if (!currentActivity || !activityTask.value || !activityMinutes.value  || !activitySeconds.value) {
+    buttonErrorMessage();
+    activityErrorMessage();
+    minutesErrorMessage();
+    secondsErrorMessage();
+  } else {
+    var activity = new Activity(currentActivity, activityTask.value, activityMinutes.value, activitySeconds.value, false);
```

Comment:
A quick note here: I wonder if its more semantic to put the new instance (var activity) at the top of the function??